### PR TITLE
HPCC-15804 ECLWatch displays partial WU XML when using Cassandra

### PR DESCRIPTION
**Pull request by richardkchapman**
Signed-off-by: Richard Chapman rchapman@hpccsystems.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@ghalliday Please review/merge

File: plugins/cassandra/cassandrawu.cpp [hunk 1]
```diff
@@ -2655,6 +2655,15 @@ class CCassandraWorkUnit : public CPersistedWorkUnit
         CPersistedWorkUnit::clearExceptions();
     }
 
+    virtual IPropertyTree *getUnpackedTree(bool includeProgress) const
+    {
+        // If anyone wants the whole ptree, we'd better make sure we have fully loaded it...
+        CriticalBlock b(crit);
+        for (const ChildTableInfo * const * table = childTables; *table != NULL; table++)
```
Comment (jakesmith) [inline plugins/cassandra/cassandrawu.cpp]: trivial: nullptr

Comment (jakesmith) [inline plugins/cassandra/cassandrawu.cpp]: Looks good to me.

**Comment by richardkchapman** [comment]
Closing in favour of https://github.com/hpcc-systems/HPCC-Platform/pull/8835

**Pull request closed without merging**